pkg/vm/manager.go

```diff
@@ -436,6 +436,7 @@ func (m *Module) Run(vm pkg.VM) (pkg.MachineInfo, error) {
 		// arguments
 		cmdline["root"] = virtioRootFsTag
 		cmdline["rootfstype"] = "virtiofs"
+		cmdline["rootdelay"] = "30"
```

Comment:
what does this value affect?

Reply:
Ah i needed it during the testing it's basically how long the init script waits for the filesystem before it's available. this is completely ignored by anything that is not using ubuntu initramfs. But i thought to keep it since it give ubuntu more time to wait before it fails, and has no effect with anything else